### DISCUSSION TITLE
Hi! Is there an example for this animation found on charm.sh?

### DISCUSSION THREAD

**Discussion by Rydez** [charmbracelet/bubbletea#880] [Q&A]
First just want to say how inspiring everything coming from charmbracelet is. Truly fascinating stuff. Really feels like the main ingredient is love ❤️. Thank you.

Anyway, I was just wondering if there's an example somewhere for how to accomplish this neato animation of little popup modals on charm.sh?
![image](https://github.com/charmbracelet/bubbletea/assets/21091975/697d6fd6-d892-4914-9b3a-d13221e3152f)

**Comment by bashbunni** [comment]
Hi @Rydez sorry for the late reply on this. I believe those examples were created in Adobe After Effects with a compositing example we were playing with. We now use VHS with After Effects when we want to add more pzazzz
